src/utils.js

```diff
@@ -179,6 +181,41 @@ const gpuPresent = async () => {
   return gpu;
 };
 
+const tfCapture = async (command, args = [], options = {}) => {
+  return new Promise((resolve, reject) => {
+    const stderrCollection = [];
+    const tfProc = require('child_process').spawn(command, args, options);
+    tfProc.stdout.on('data', (buf) => {
+      const parse = (line) => {
+        if (line === '') return;
+        try {
+          const log = JSON.parse(line);
+          if (log['@level'] === 'error') {
+            winston.error(`terraform error: ${log['@message']}`);
+          } else {
+            winston.info(log['@message']);
+          }
```

Comment:
`-Wperhaps-it-is-more-clear`

```suggestion
          const { '@level': level, '@message': message } = JSON.parse(line);
          if (level === 'error') {
            winston.error(`terraform error: ${message}`);
          } else {
            winston.info(message);
          }
```

Reply:
we can get one more opinion @casperdcl / @DavidGOrtega ?

Reply:
always deestructure